roles/edpm_nova/templates/nova_compute.json.j2

```diff
@@ -13,7 +13,7 @@
 {% if edpm_nova_tls_certs_enabled %}
 	"/var/lib/openstack/certs/nova/tls.crt:/etc/pki/nova/server-cert.pem:ro",
 	"/var/lib/openstack/certs/nova/tls.key:/etc/pki/nova/private/server-key.pem:ro",
-	"/var/lib/openstack/cacerts/nova/TLSCABundleFile:/etc/pki/CA/cacert.pem:ro",
+	"/var/lib/openstack/cacerts/nova/tls-ca-bundle.pem:/etc/pki/CA/cacert.pem:ro",
```

Comment:
nova will need the ca key so that it will trust the libvirt certs although I'm not sure if we actually need client certs for nova itself since we are not using two way cert auth